kinto/core/resource/viewset.py

```diff
@@ -118,14 +128,17 @@ def get_view_arguments(self, endpoint_type, resource_cls, method):
         endpoint_args = getattr(self, by_method, {})
         args.update(**endpoint_args)
 
+        request_schema = args.get('schema', RequestSchema)
+
+        # Instantiate request schema
+        request_schema = request_schema()
+
         if method.lower() in map(str.lower, self.validate_schema_for):
```

Comment:
Note: is this `validate_schema_for` still relevant with the changes of this PR? Maybe we can get rid of it.